modules/tinymce/src/core/main/ts/schema/ValidChildrenRuleParser.ts

```diff
@@ -29,17 +38,9 @@ export const parseValidChildrenRules = (value: string): ValidChildrenRule[] => {
       const prefix = matches[1];
       const operation = prefix ? prefixToOperation(prefix) : 'replace';
       const name = matches[2];
-      const validChildren = Arr.bind(SchemaUtils.split(matches[3], '|'), (validChild) => {
-        const nameMatches = validChildRegExp.exec(validChild);
-        if (nameMatches) {
-          const preset = nameMatches[1] === '@';
-          const name = nameMatches[2];
-
-          return [{ preset, name }];
-        } else {
-          return [];
-        }
-      });
+      const validChildren = Arr.bind(SchemaUtils.split(matches[3], '|'), (validChild) =>
+        parseValidChild(validChild).fold(Fun.constant([]), Arr.pure)
```

Comment:
I think this is equivalent
```suggestion
        parseValidChild(validChild).toArray();
```

Reply:
Right forgot about that one.